schema.sql

```diff
@@ -67,3 +67,17 @@ CREATE TABLE visits (
    FOREIGN KEY (animal_id) REFERENCES animals(id),
    FOREIGN KEY (vet_id) REFERENCES vetS(id)
 );
+
+--  It is important to keep only the queries that change the database structure in the schema.sql file.
+
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+CREATE INDEX animal_id_desc
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+
+explain analyze SELECT * FROM visits where vet_id = 2;
+CREATE INDEX vet_index
+explain analyze SELECT * FROM visits where vet_id = 2;
+
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+CREATE INDEX email_desc ON owners(email DESC);
```

Comment:
- Good job so far. Please note the you should include the screenshots in the PR description with the results of the  EXPLAIN ANALYZE command for each of the above queries (in total it should be 6 screenshots):

The first screenshot should show how much time the query takes BEFORE the improvements introduced,
- The execution time should be around 1 sec (1000 ms).
The second screenshot should show how much time the query takes AFTER the improvements introduced
- The execution time should be significantly lower than 1 sec (1000 ms).👍🏾

Reply:
Hi @rotimiazeez, Kindly find the screenshots as requested in the project requirement